readme.md

```diff
@@ -6,6 +6,16 @@ TypeScript client library for [Jellyfish](https://github.com/jellyfish-dev/jelly
 
 Documentation is available [here](https://jellyfish-dev.github.io/ts-client-sdk/)
 
+## Installation
+
+You can install this package using `npm`:
+
+```bash
+npm https://github.com/jellyfish-dev/ts-client-sdk
```

Comment:
But we should encourage people to install released version. Github installation is only four our local tests, we should make a new release

You can install this package using `npm`:
```shell
npm install @jellyfish-dev/ts-client-sdk
```